lib/crudry_query.ex

```diff
@@ -84,4 +81,10 @@ defmodule Crudry.Query do
         )
     end)
   end
+
+  defp parse_order_by_args(_, order_by) when is_list(order_by), do: order_by
```

Comment:
What if it's a list of strings? Do you think we should support that?

Reply:
We are not going to use it right now, so no need to implement it.